src/core/dom/image/test/index.js

```diff
@@ -139,6 +139,33 @@ module.exports = async (page, params) => {
 					expect(await getNode(tag).evaluate((ctx) => globalThis.getSrc(ctx))).toBe(images.pngImage);
 				});
 
+				it('with `src` and optionsResolver', async () => {
```

Comment:
Если в описании ссылаешьс на метод или свойство, то бери его в апострофы

```suggestion
				it('with `src` and `optionsResolver`', async () => {
```